jenkins/spark-premerge-build.sh

```diff
@@ -111,6 +111,13 @@ MVN_GET_CMD="mvn org.apache.maven.plugins:maven-dependency-plugin:2.8:get -B \
 
 rm -rf $ARTF_ROOT && mkdir -p $ARTF_ROOT
 
+# If possible create '~/.m2' cache from pre-created m2 tarball to minimize the impact of unstable network connection.
+# Please refer to job 'update_premerge_m2_cache' on Blossom about building m2 tarball details.
+M2_CACHE_TAR=${M2_CACHE_TAR:-"/home/jenkins/agent/m2_cache/premerge_m2_cache.tar"}
```

Comment:
Where can we find the contents of this tar archive?  Is there a jenkins job responsible for creating the artifact?  It would be nice for the comments here to point people in the right direction to learn more about it in case it needs to be fixed or updated as dependencies change.

Reply:
yes, there's Jenkins job `update_premerge_m2_cache` on Blossom. Since it's internal, how about add some description to the document about the content?

Reply:
If we can't link to it directly then mentioning the job name is fine.  The danger of being too detailed here is that someone could update the Blossom job, invalidating the description here, and they forget to update it.  I think mentioning the Blossom job name is fine.

Reply:
I see. Let met add some comments to the job on Blossom and corresponding Jenkinsfile in order to minimize such issue.

Reply:
Blossom job and Jenkinsfile are updated. Please refer to internal Blossom system